src/apps/front-office/account/pages/register-page/RegisterPage.tsx

```diff
@@ -0,0 +1,121 @@
+import { Form } from "@mongez/react-form";
+import { Link, navigateBack } from "@mongez/react-router";
+import { BookOpen } from "lucide-react";
+import Footer from "src/apps/front-office/design-system/layouts/Footer/Footer";
+import Header from "src/apps/front-office/design-system/layouts/Header/Header";
+import Checkbox from "./components/Checkbox";
+import EmailInput from "./components/EmailInput";
+import HeadOfRegister from "./components/HeadOfRegister";
+import PasswordInput from "./components/PasswordInput";
+import RadioGroup from "./components/RadioGroup";
+import RadioInput from "./components/RadioInput";
+import SocialLogin from "./components/SocialLogin";
+import TextInput from "./components/TextInput";
+import "./input.css";
+
+export default function RegisterPage() {
+  const submitForm = ({ values, form }) => {
```

Comment:
It would be better if we moved the logic part into a hook, also use `endpoint` instead of fetch please